docs/cmake.md

```diff
@@ -158,76 +181,86 @@ The following assumes Visual Studio 2017 using 64-bit architecture (adjust the
 CMake `-G` parameter if using something different).
 
 ```bat
-cd tests\cmake\Messenger\Messenger_1
+cd DevGuideExamples\DCPS\Messenger
 mkdir build
 cd build
-cmake -DCMAKE_PREFIX_PATH=<location of OpenDDS source tree> -G "Visual Studio 15 2017 Win64" ..
+cmake -G "Visual Studio 15 2017 Win64" ..
 cmake --build .
+perl run_test.pl
 ```
 
-### Example Using Installed OpenDDS (Unix only)
+#### Example Using Installed OpenDDS (Unix only)
 
 The `--prefix` switch can be passed to configure to enable the `install` target,
 which will install OpenDDS (including the OpenDDS CMake Config module) into the
-specified directory.  See the [INSTALL](../INSTALL) document for details.
+specified directory. See the [INSTALL.md](../INSTALL.md) document for details.
 
 *Note:* Be sure to pass an absolute path to `--prefix`.
 
 ```bash
-DDS_WORKSPACE=$(pwd)
-cd OpenDDS-src
-./configure --prefix=$DDS_WORKSPACE/opendds-install
-make
+OPENDDS_PREFIX="$PWD/opendds-install"
+DDS_ROOT="$PWD/OpenDDS"
+ACE_ROOT="$DDS_ROOT/ACE_wrappers"
+export PERL5LIB="$DDS_ROOT/bin:$ACE_ROOT/bin"
```

Comment:
Is this needed for the build or could we move it down to right before (or with) the `run_test.pl` line?